crates/cheatcodes/src/inspector.rs

```diff
@@ -348,6 +348,17 @@ pub struct Cheatcodes {
     /// Use ZK-VM to execute CALLs and CREATEs.
     pub use_zk_vm: bool,
 
+    /// When in zkEVM context, execute the next CALL or CREATE in the EVM instead.
+    pub skip_zk_vm: bool,
+
+    /// Any contracts that were deployed in `skip_zk_vm` step.
+    /// This makes it easier to dispatch calls to any of these addresses in zkEVM context, directly
+    /// to EVM. Alternatively, we'd need to add `vm.zkVmSkip()` to these calls manually.
+    pub skip_zk_vm_addresses: HashSet<Address>,
```

Comment:
Do you think it's worth to add an extra cheatcode (or a specialization of it) to insert an address manually?

Reply:
That would be risky, if anything I wanted to have this field private but that causes instantiation issues.

Reply:
Hmm okay, I was thinking as a manual override or as part of a contract setup

Reply:
Let's wait until there's a proper use case before we add more custom cheatcodes